### PR TITLE
Armstrong number code in Kotlin

### PR DESCRIPTION
**Pull request by ritvij14**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Code for checking Armstrong number in Kotlin
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Issue #2262 
(Write your answer here.)

### CODE REVIEW

**Comment by ritvij14** [comment]
@HarshCasper @siddharth25pandey @iamrajiv please review this PR.

File: Kotlin/Maths/Armstrong.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+fun isArmstrong(num: Int): Boolean {
+    var isArmNum = false
+    var result : Int= 0
+    var temp = num
+
+    // Count No Digits in numbers
+    val digits = temp.toString().length
+
+    while (temp > 0) {
+        val r = temp % 10
```
Comment (gat786) [inline Kotlin/Maths/Armstrong.kt]: you should rename `r` as remainder just as to make it more readable

**Review by gat786 — COMMENTED** [review]
Code looks good to me just do the change I have requested and I will merge it

File: Kotlin/Maths/Armstrong.kt (hunk 1 above)
Comment (ritvij14) [reply Kotlin/Maths/Armstrong.kt]: @gat786 changes done. Please review.

File: Kotlin/Maths/Armstrong.kt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import java.util.*
+
+// Function for check Armstrong Number
+fun isArmstrong(num: Int): Boolean {
+    var isArmNum = false
+    var result : Int= 0
+    var temp = num
+
+    // Count No Digits in numbers
+    val digits = temp.toString().length
```
Comment (gat786) [inline Kotlin/Maths/Armstrong.kt]: can you rename this to `numberOfDigits`. Sorry for pointing out one things at a time.

Comment (gat786) [reply Kotlin/Maths/Armstrong.kt]: Man, I had left this comment a day ago, but I forgot to click on comment button. Sorry for the delay in messaging. And thanks for contributing and taking your time out.

Comment (ritvij14) [reply Kotlin/Maths/Armstrong.kt]: @gat786 no problems! :)
The change is done.

**Comment by gat786** [comment]
YaY 🔥🎊

**Comment by ritvij14** [comment]
@gat786 please add a label according to this table so that it gets counted in the leaderboard.
image

**Comment by gat786** [comment]
Ah sorry ritvij I didn't knew about the working of points and assigning labels. Thanks for contributing tho.

**Comment by ritvij14** [comment]
@gat786 thanks for adding the tag, I also came to know about it today only.

**Pull request merged**